docs/en/sql-reference/table-functions/values.md

```diff
@@ -0,0 +1,189 @@
+---
+slug: /sql-reference/table-functions/values
+sidebar_position: 210
+sidebar_label: values
+title: 'values'
+description: 'creates a temporary storage which fills columns with values.'
+keywords: ['values', 'table function']
+---
+
+# Values Table Function {#values-table-function}
+
+The `Values` table function allows you to create temporary storage which fills 
+columns with values. It is useful for quick testing or generating sample data.
+
+:::note
+Values is a case-insensitive function. I.e. `VALUES` or `values` are both valid.
+:::
+
+## Syntax {#syntax}
+
+The basic syntax of the `VALUES` table function is:
+
+```sql
+VALUES([structure,] values...)
+```
+
+It is commonly used as:
+
+```sql
+VALUES(
+    ['column1_name Type1, column2_name Type2, ...'],
+    (value1_row1, value2_row1, ...),
+    (value1_row2, value2_row2, ...),
+    ...
+)
+```
+
+## Arguments {#arguments}
+
+- `column1_name Type1, ...` (optional). [String](/sql-reference/data-types/string) 
+  specifying the column names and types. If this argument is omitted columns will
+  be named as `c1`, `c2`, etc.
+- `(value1_row1, value2_row1)`. [Tuples](/sql-reference/data-types/tuple) 
+   containing values of any type.
+
+:::note
+Comma separated tuples can be replaced by single values as well. In this case
+each value is taken to be a new row. See the [examples](#examples) section for
+details.
+:::
+
+## Returned value {#returned-value}
+
+- Returns a temporary table containing the provided values.
+
+## Examples {#examples}
+
+```sql title="Query"
+SELECT *
+FROM VALUES(
+    'person String, place String',
+    ('Noah', 'Paris'),
+    ('Emma', 'Tokyo'),
+    ('Liam', 'Sydney'),
+    ('Olivia', 'Berlin'),
+    ('Ilya', 'London'),
+    ('Sophia', 'London'),
+    ('Jackson', 'Madrid'),
+    ('Alexey', 'Amsterdam'),
+    ('Mason', 'Venice'),
+    ('Isabella', 'Prague')
+)
+```
+
+```response
+    ┌─person───┬─place─────┐
+ 1. │ Noah     │ Paris     │
+ 2. │ Emma     │ Tokyo     │
+ 3. │ Liam     │ Sydney    │
+ 4. │ Olivia   │ Berlin    │
+ 5. │ Ilya     │ London    │
+ 6. │ Sophia   │ London    │
+ 7. │ Jackson  │ Madrid    │
+ 8. │ Alexey   │ Amsterdam │
+ 9. │ Mason    │ Venice    │
+10. │ Isabella │ Prague    │
+    └──────────┴───────────┘
+```
+
+`VALUES` can also be used with single values rather than tuples. For example:
+
+```sql title="Query"
+FROM VALUES(
+    'person String',
+    'Noah',
+    'Emma',
+    'Liam',
+    'Olivia',
+    'Ilya',
+    'Sophia',
+    'Jackson',
+    'Alexey',
+    'Mason',
+    'Isabella'
+)
+```
+
+```response title="Response"
```

Comment:
`nit`: add `title="Response"` to other response blocks as well? The one above and below and missing it/